### ISSUE TITLE
Building hackrf host code w/out fftw

### ISSUE THREAD

**Issue by jjamison456** [mossmann/hackrf#729]
For my application, it would be nice if there was a way to build the hackrf host code without the fftw3 requirement.  Is this possible?  I'm guessing this is only used by the sweep mode, so maybe there is a cmake flag I am missing to disable this mode and this requirement.

If this doesn't exist I think this would be a helpful option to have if it only disables some functionality (i.e. the sweep mode).

**Comment by miek** [comment]
No, it's not possible to build without fftw right now. I'll leave this open in case we want to add that option in the future.

**Comment by martinling** [comment]
Fixed in #1586.
